Components/Avatar/avatar.css

```diff
@@ -0,0 +1,32 @@
+* {
```

Comment:
if it's for a component, it should be in a class.
If it's general, it should be in a general css file